spacemesh/v2/account.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package spacemesh.v2;
+
+message Account {
+  oneof versioned {
+    AccountV1 v1 = 1;
+  }
+}
+
+message AccountV1 {
+  string account_id = 1; // account public address
+  AccountState state_current = 2; // current state
+  AccountState state_projected = 3; // projected state (includes pending txs)
+}
+
+message AccountState {
+  uint64 counter = 1; // aka nonce
+  uint64 balance = 2; // known account balance
+}
+
+message AccountRequest {
+  string address = 1;
```

Comment:
I think it will be more user friendly to use string, and we already use string in V1. https://github.com/spacemeshos/api/blob/master/spacemesh/v1/types.proto#L19